objc-class-ref in GeneratedPluginRegistrant.o ld: symbol(s) not found for architecture x86_64

## Steps to Reproduce

<!-- You must include full steps to reproduce so that we can reproduce the problem. -->

1. Run `flutter build ios`.
2. Open iOS in XCode
3. Build Project

**Expected results:**
Application Builds

**Actual results:** <!-- what did you see? -->
XCode Build Failed:

<details>
  <summary>Logs</summary>



```

Showing Recent Issues

Build target Runner of project Runner with configuration Debug

Ld /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/Tempo.app/Tempo normal x86_64 (in target 'Runner' from project 'Runner')
    cd /Users/admin/StudioProjects/tempo-guided-breathing/ios
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target x86_64-apple-ios8.0-simulator -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk -L/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator -L/Users/admin/StudioProjects/tempo-guided-breathing/ios/Flutter -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/AppAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/BoringSSL-GRPC -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseCore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseCoreDiagnostics -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseFirestore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseInstallations -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseInstanceID -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseMessaging -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseStorage -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GTMAppAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GTMSessionFetcher -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleDataTransport -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleDataTransportCCTSupport -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleUtilities -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/PromisesObjC -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/Protobuf -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/PureLayout -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/abseil -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/audioplayers -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/cloud_firestore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/file_picker -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_admob -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_analytics -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_auth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_core -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_messaging -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_storage -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/flutter_native_admob -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/gRPC-C++ -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/gRPC-Core -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/google_sign_in -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/leveldb-library -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/nanopb -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/path_provider -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/share -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/url_launcher -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/vibration -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/wakelock -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/../Flutter -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/FirebaseAnalytics/Frameworks -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/GoogleAppMeasurement/Frameworks -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/GoogleSignIn/Frameworks -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/AppAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/BoringSSL-GRPC -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseCore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseCoreDiagnostics -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseFirestore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseInstallations -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseInstanceID -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseMessaging -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/FirebaseStorage -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GTMAppAuth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GTMSessionFetcher -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleDataTransport -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleDataTransportCCTSupport -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/GoogleUtilities -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/PromisesObjC -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/Protobuf -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/PureLayout -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/abseil -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/audioplayers -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/cloud_firestore -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/file_picker -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_admob -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_analytics -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_auth -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_core -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_messaging -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/firebase_storage -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/flutter_native_admob -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/gRPC-C++ -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/gRPC-Core -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/google_sign_in -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/leveldb-library -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/nanopb -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/path_provider -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/share -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/url_launcher -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/vibration -F/Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/wakelock -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/../Flutter -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/FirebaseAnalytics/Frameworks -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/GoogleAppMeasurement/Frameworks -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Pods/GoogleSignIn/Frameworks -F/Users/admin/StudioProjects/tempo-guided-breathing/ios/Flutter -filelist /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Intermediates.noindex/Runner.build/Debug-iphonesimulator/Runner.build/Objects-normal/x86_64/Tempo.LinkFileList -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Intermediates.noindex/Runner.build/Debug-iphonesimulator/Runner.build/Objects-normal/x86_64/Tempo_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Intermediates.noindex/Runner.build/Debug-iphonesimulator/Runner.build/Objects-normal/x86_64/Tempo.swiftmodule -ObjC -lc++ -lsqlite3 -lz -framework AVFoundation -framework AppAuth -framework AudioToolbox -framework CFNetwork -framework CoreGraphics -framework CoreMedia -framework CoreTelephony -framework CoreText -framework CoreVideo -framework FBLPromises -framework FIRAnalyticsConnector -framework FirebaseAnalytics -framework FirebaseAuth -framework FirebaseCore -framework FirebaseCoreDiagnostics -framework FirebaseFirestore -framework FirebaseInstallations -framework FirebaseInstanceID -framework FirebaseMessaging -framework FirebaseStorage -framework Flutter -framework Foundation -framework GTMAppAuth -framework GTMSessionFetcher -framework GoogleAppMeasurement -framework GoogleDataTransport -framework GoogleDataTransportCCTSupport -framework GoogleMobileAds -framework GoogleSignIn -framework GoogleUtilities -framework LocalAuthentication -framework MediaPlayer -framework MessageUI -framework MobileCoreServices -framework PureLayout -framework QuartzCore -framework SafariServices -framework Security -framework StoreKit -framework SystemConfiguration -framework UIKit -framework absl -framework audioplayers -framework cloud_firestore -framework file_picker -framework firebase_admob -framework firebase_analytics -framework firebase_auth -framework firebase_core -framework firebase_messaging -framework firebase_storage -framework flutter_native_admob -framework google_sign_in -framework grpc -framework grpcpp -framework leveldb -framework nanopb -framework openssl_grpc -framework path_provider -framework protobuf -framework share -framework url_launcher -framework vibration -framework wakelock -weak_framework AdSupport -weak_framework AuthenticationServices -weak_framework JavaScriptCore -weak_framework SafariServices -weak_framework UserNotifications -weak_framework WebKit -framework Flutter -ObjC -lc++ -lsqlite3 -lz -framework AVFoundation -framework AppAuth -framework AudioToolbox -framework CFNetwork -framework CoreGraphics -framework CoreMedia -framework CoreTelephony -framework CoreText -framework CoreVideo -framework FBLPromises -framework FIRAnalyticsConnector -framework FirebaseAnalytics -framework FirebaseAuth -framework FirebaseCore -framework FirebaseCoreDiagnostics -framework FirebaseFirestore -framework FirebaseInstallations -framework FirebaseInstanceID -framework FirebaseMessaging -framework FirebaseStorage -framework Flutter -framework Foundation -framework GTMAppAuth -framework GTMSessionFetcher -framework GoogleAppMeasurement -framework GoogleDataTransport -framework GoogleDataTransportCCTSupport -framework GoogleMobileAds -framework GoogleSignIn -framework GoogleUtilities -framework LocalAuthentication -framework MediaPlayer -framework MessageUI -framework MobileCoreServices -framework PureLayout -framework QuartzCore -framework SafariServices -framework Security -framework StoreKit -framework SystemConfiguration -framework UIKit -framework absl -framework audioplayers -framework cloud_firestore -framework file_picker -framework firebase_admob -framework firebase_analytics -framework firebase_auth -framework firebase_core -framework firebase_messaging -framework firebase_storage -framework flutter_native_admob -framework google_sign_in -framework grpc -framework grpcpp -framework leveldb -framework nanopb -framework openssl_grpc -framework path_provider -framework protobuf -framework share -framework url_launcher -framework vibration -framework wakelock -weak_framework AdSupport -weak_framework AuthenticationServices -weak_framework JavaScriptCore -weak_framework SafariServices -weak_framework UserNotifications -weak_framework WebKit -framework Flutter -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Intermediates.noindex/Runner.build/Debug-iphonesimulator/Runner.build/Tempo.app-Simulated.xcent -framework Pods_Runner -Xlinker -dependency_info -Xlinker /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Intermediates.noindex/Runner.build/Debug-iphonesimulator/Runner.build/Objects-normal/x86_64/Tempo_dependency_info.dat -o /Users/admin/Library/Developer/Xcode/DerivedData/Runner-fdiewvaqiesrjoggcmbthlijahou/Build/Products/Debug-iphonesimulator/Tempo.app/Tempo

C, referenced from:
      objc-class-ref in GeneratedPluginRegistrant.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)


```

<!--
     Run `flutter analyze` and attach any output of that command below.
     If there are any analysis errors, try resolving them before filing this issue.
-->

```
admin@DM108 ios % flutter analyze
Analyzing ios...                                                        
No issues found! (ran in 4.0s)
admin@DM108 ios % 
```

<!-- Finally, paste the output of running `flutter doctor -v` here. -->

```

[✓] Flutter (Channel master, v1.15.4-pre.267, on Mac OS X 10.15.3 19D76, locale en-US)
    • Flutter version 1.15.4-pre.267 at /Users/admin/Desktop/flutter
    • Framework revision 0a84024b86 (24 hours ago), 2020-03-03 23:19:01 -0500
    • Engine revision 90bd6f8148
    • Dart version 2.8.0 (build 2.8.0-dev.11.0 5701c4fd3b)

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/admin/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit https://flutter.dev/setup/#android-setup for detailed instructions.

 
[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.9.0

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • ECA13ABE-B1F0-47D3-BDFF-8A3BE2462172 • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-3 (simulator)

! Doctor found issues in 1 category.
admin@DM108 tempo-guided-breathing % 

```

</details>

I'm also seeing this when switching from `stable` to `master`

```
 Undefined symbols for architecture arm64:
      "_OBJC_CLASS_$_GULSecureCoding", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessagingPubSub.o)
      "_GULLoggerRegisterVersion", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_GULLoggerInitializeASL", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULHeartbeatDateStorage", referenced from:
          objc-class-ref in FirebaseCore(FIRHeartbeatInfo.o)
      "_GULLogBasic", referenced from:
          _FIRLogBasic in FirebaseCore(FIRLogger.o)
      "_GULLoggerEnableSTDERR", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULUserDefaults", referenced from:
          objc-class-ref in FirebaseInstanceID(FIRInstanceID.o)
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDUtilities.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingPubSub.o)
          objc-class-ref in FirebaseInstallations(FIRInstallationsStore.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDCheckinPreferences.o)
      "_GULLoggerForceDebug", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_GULSetLoggerLevel", referenced from:
          _FIRSetLoggerLevel in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULAppDelegateSwizzler", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingAnalytics.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingContextManagerService.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingRemoteNotificationsProxy.o)
      "_OBJC_CLASS_$_GULReachabilityChecker", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
      "_OBJC_CLASS_$_GULAppEnvironmentUtil", referenced from:
          objc-class-ref in FirebaseCore(FIRApp.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceID.o)
          objc-class-ref in FirebaseCore(FIRBundleUtil.o)
      "_OBJC_CLASS_$_CardIOPaymentViewController", referenced from:
          objc-class-ref in flutter_card_io(FlutterCardIoPlugin.o)
    ld: symbol(s) not found for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

This is how I reproduce by just switching to master

Calvins-MacBook-Pro:~ calvinmuller$ `flutter channel master`
Switching to flutter channel 'master'...
git: Switched to branch 'master'
Calvins-MacBook-Pro:donatenation calvinmuller$ `flutter run`
```
Downloading Dart SDK from Flutter engine 2e2a9e192d3d8f4f8aa96ca283947f4125bf836c...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  211M  100  211M    0     0  4664k      0  0:00:46  0:00:46 --:--:-- 4817k
Building flutter tool...
Waiting for another flutter command to release the startup lock...
Downloading ios tools...                                            9,3s
Downloading ios-profile tools...                                    7,4s
Downloading ios-release tools...                                   37,1s
Downloading Web SDK...                                              2,8s
Downloading darwin-x64/FlutterMacOS.framework tools...              4,1s
Downloading darwin-x64-profile/FlutterMacOS.framework tools...         2,5s
Downloading darwin-x64-profile tools...                             2,0s
Downloading darwin-x64-release/FlutterMacOS.framework tools...         2,2s
Downloading darwin-x64-release tools...                             1,8s
```

```
Running "flutter pub get" in donatenation...                        1,6s
Launching lib/main.dart on iPhone 11 Pro in debug mode...
 
Running pod install...                                              4,6s
Running Xcode build...                                                  
                                                   
 └─Compiling, linking and signing...                         2,9s
Xcode build done.                                           20,3s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    === BUILD TARGET yp_image_picker OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/calvinmuller/.pub-cache/git/flutter_card_io-6566879ce6d40e690355f1eb101df30c78df40c0/ios/Classes/FlutterCardIoPlugin.m:44:38: warning: assigning to 'id<UINavigationControllerDelegate> _Nullable' from incompatible type
    'FlutterCardIoPlugin *const __strong'
            _scanViewController.delegate = self;
                                         ^ ~~~~
    1 warning generated.
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/local_auth-0.6.1+3/ios/Classes/FLTLocalAuthPlugin.m:117:33: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
                                    lastCallArgs = arguments;
                                    ^
                                    self->
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/local_auth-0.6.1+3/ios/Classes/FLTLocalAuthPlugin.m:118:33: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior
    [-Wimplicit-retain-self]
                                    lastResult = result;
                                    ^
                                    self->
    2 warnings generated.
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/sqflite-1.2.1/ios/Classes/SqflitePlugin.m:2:9: warning: non-portable path to file '<fmdb/FMDB.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
    #import <FMDB/FMDB.h>
            ^~~~~~~~~~~~~
            <fmdb/FMDB.h>
    1 warning generated.
    In file included from /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:1:
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    + (instancetype)sharedInstance;
       ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the pointer may be null
    + (instancetype)sharedInstance;
       ^
                    _Nullable 
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    + (instancetype)sharedInstance;
       ^
                    _Nonnull 
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: warning: block pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)
    [-Wnullability-completeness]
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the block pointer may be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nullable
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the block pointer should never be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nonnull
    2 warnings generated.
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:193:43: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for
    downstream message handling. [-Wdeprecated-declarations]
    - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                                              ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:277:24: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for
    downstream message handling. [-Wdeprecated-declarations]
        didReceiveMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                           ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:140:30: warning: 'shouldEstablishDirectChannel' is deprecated: FCM direct channel is deprecated, please use APNs
    channel for downstream message delivery. [-Wdeprecated-declarations]
        [FIRMessaging messaging].shouldEstablishDirectChannel = true;
                                 ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:283:56: note: 'shouldEstablishDirectChannel' has been explicitly marked deprecated here
    @property(nonatomic) BOOL shouldEstablishDirectChannel DEPRECATED_MSG_ATTRIBUTE(
                                                           ^
    In module 'UIKit' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/usr/include/AvailabilityMacros.h:182:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:194:52: warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [self didReceiveRemoteNotification:remoteMessage.appData];
                                                       ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:276:1: warning: implementing deprecated method [-Wdeprecated-implementations]
    - (void)messaging:(FIRMessaging *)messaging
    ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:253:1: note: method 'messaging:didReceiveMessage:' declared here
    - (void)messaging:(FIRMessaging *)messaging
    ^
    /Users/calvinmuller/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.12/ios/Classes/FLTFirebaseMessagingPlugin.m:278:63: warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [_channel invokeMethod:@"onMessage" arguments:remoteMessage.appData];
                                                                  ^
    In module 'FirebaseMessaging' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.2.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    6 warnings generated.
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
    The use of Swift 3 @objc inference in Swift 4 mode is deprecated. Please address deprecated @objc inference warnings, test your code with “Use of deprecated Swift 3 @objc inference” logging enabled, and then disable inference by changing
    the "Swift 3 @objc Inference" build setting to "Default" for the "Runner" target.
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:85:34: warning: expression implicitly coerced from 'UIImage?' to 'Any'
                        items.append(image)
                                     ^~~~~
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:85:34: note: provide a default value to avoid this warning
                        items.append(image)
                                     ^~~~~
                                           ?? <#default value#>
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:85:34: note: force-unwrap the value to avoid this warning
                        items.append(image)
                                     ^~~~~
                                          !
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:85:34: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        items.append(image)
                                     ^~~~~
                                           as Any
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:97:33: warning: expression implicitly coerced from 'String?' to 'Any'
                        "activity": activity?.rawValue
                                    ^~~~~~~~~~~~~~~~~~
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:97:43: note: provide a default value to avoid this warning
                        "activity": activity?.rawValue
                                    ~~~~~~~~~~^~~~~~~~
                                                       ?? <#default value#>
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:97:43: note: force-unwrap the value to avoid this warning
                        "activity": activity?.rawValue
                                    ~~~~~~~~~~^~~~~~~~
                                                      !
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:97:43: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "activity": activity?.rawValue
                                    ~~~~~~~~~~^~~~~~~~
                                                       as Any
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/SharePlugin.swift:102:11: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
            } catch let error {
              ^
    /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/Plugins/WebviewPlugin.swift:47:34: warning: 'openURL' was deprecated in iOS 10.0
                UIApplication.shared.openURL(NSURL(string:text)! as URL)
                                     ^
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
    While building module 'uni_links' imported from /Users/calvinmuller/iStreet/Mobile/donatenation/ios/Runner/GeneratedPluginRegistrant.m:62:
    In file included from <module-includes>:1:
    In file included from /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/uni_links-umbrella.h:13:
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or
    _Null_unspecified) [-Wnullability-completeness]
    + (instancetype)sharedInstance;
       ^
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the pointer may be null
    + (instancetype)sharedInstance;
       ^
                    _Nullable 
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    + (instancetype)sharedInstance;
       ^
                    _Nonnull 
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:7:33: warning: block pointer is missing a nullability type specifier (_Nonnull, _Nullable, or
    _Null_unspecified) [-Wnullability-completeness]
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the block pointer may be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nullable
    /Users/calvinmuller/iStreet/Mobile/donatenation/build/ios/Debug-iphonesimulator/uni_links/uni_links.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the block pointer should never be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nonnull
    2 warnings generated.
    2 warnings generated.
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_GULSecureCoding", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessagingPubSub.o)
      "_OBJC_CLASS_$_GULHeartbeatDateStorage", referenced from:
          objc-class-ref in FirebaseCore(FIRHeartbeatInfo.o)
      "_GULLogBasic", referenced from:
          _FIRLogBasic in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULAppEnvironmentUtil", referenced from:
          objc-class-ref in FirebaseCore(FIRApp.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceID.o)
          objc-class-ref in FirebaseCore(FIRBundleUtil.o)
      "_GULResetLogger", referenced from:
          _FIRResetLogger in FirebaseCore(FIRLogger.o)
      "_GULLoggerRegisterVersion", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_GULLoggerInitializeASL", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_GULLoggerEnableSTDERR", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULUserDefaults", referenced from:
          objc-class-ref in FirebaseInstanceID(FIRInstanceID.o)
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDUtilities.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingPubSub.o)
          objc-class-ref in FirebaseInstallations(FIRInstallationsStore.o)
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDCheckinPreferences.o)
      "_GULLoggerForceDebug", referenced from:
          ___FIRLoggerInitializeASL_block_invoke in FirebaseCore(FIRLogger.o)
      "_GULSetLoggerLevel", referenced from:
          _FIRSetLoggerLevel in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_GULAppDelegateSwizzler", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingAnalytics.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingContextManagerService.o)
          objc-class-ref in FirebaseMessaging(FIRMessagingRemoteNotificationsProxy.o)
      "_OBJC_CLASS_$_GULReachabilityChecker", referenced from:
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)
      "_GULIsLoggableLevel", referenced from:
          _FIRIsLoggableLevel in FirebaseCore(FIRLogger.o)
      "_OBJC_CLASS_$_CardIOPaymentViewController", referenced from:
          objc-class-ref in flutter_card_io(FlutterCardIoPlugin.o)
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro.

```

--- a/packages/flutter_tools/lib/src/ios/mac.dart
+++ b/packages/flutter_tools/lib/src/ios/mac.dart
@@ -24,6 +24,7 @@ import '../reporting/reporting.dart';
 import 'code_signing.dart';
 import 'migrations/ios_migrator.dart';
 import 'migrations/remove_framework_link_and_embedding_migration.dart';
+import 'migrations/xcode_build_system_migration.dart';
 import 'xcodeproj.dart';
 
 class IMobileDevice {
@@ -90,7 +91,8 @@ Future<XcodeBuildResult> buildXcodeProject({
   }
 
   final List<IOSMigrator> migrators = <IOSMigrator>[
-    RemoveFrameworkLinkAndEmbeddingMigration(app.project, globals.logger, globals.xcode, globals.flutterUsage)
+    RemoveFrameworkLinkAndEmbeddingMigration(app.project, globals.logger, globals.xcode, globals.flutterUsage),
+    XcodeBuildSystemMigration(app.project, globals.logger),
   ];
 
   final IOSMigration migration = IOSMigration(migrators);

--- a/packages/flutter_tools/lib/src/ios/migrations/ios_migrator.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/ios_migrator.dart
@@ -20,7 +20,10 @@ abstract class IOSMigrator {
   bool migrate();
 
   /// Return null if the line should be deleted.
-  String migrateLine(String line);
+  @protected
+  String migrateLine(String line) {
+    return line;
+  }
 
   @protected
   void processFileLines(File file) {

--- a/packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart
@@ -31,10 +31,6 @@ class RemoveFrameworkLinkAndEmbeddingMigration extends IOSMigrator {
   /// Inspect [project] for necessary migrations and rewrite files as needed.
   @override
   bool migrate() {
-    return _migrateXcodeProjectInfoFile();
-  }
-
-  bool _migrateXcodeProjectInfoFile() {
     if (!_xcodeProjectInfoFile.existsSync()) {
       logger.printTrace('Xcode project not found, skipping migration');
       return true;

--- a/packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart
@@ -10,9 +10,9 @@ import '../../project.dart';
 import '../../reporting/reporting.dart';
 import 'ios_migrator.dart';
 
-/// Xcode 11.4 requires linked and embedded frameworks to contain all targeted architectures before build phases are run.
-/// This caused issues switching between a real device and simulator due to architecture mismatch.
-/// Remove the linking and embedding logic from the Xcode project to give the tool more control over these.
+// Xcode 11.4 requires linked and embedded frameworks to contain all targeted architectures before build phases are run.
+// This caused issues switching between a real device and simulator due to architecture mismatch.
+// Remove the linking and embedding logic from the Xcode project to give the tool more control over these.
 class RemoveFrameworkLinkAndEmbeddingMigration extends IOSMigrator {
   RemoveFrameworkLinkAndEmbeddingMigration(
     IosProject project,
@@ -28,7 +28,6 @@ class RemoveFrameworkLinkAndEmbeddingMigration extends IOSMigrator {
   final Xcode _xcode;
   final Usage _usage;
 
-  /// Inspect [project] for necessary migrations and rewrite files as needed.
   @override
   bool migrate() {
     if (!_xcodeProjectInfoFile.existsSync()) {

--- a/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
@@ -30,13 +30,11 @@ class XcodeBuildSystemMigration extends IOSMigrator {
 
     // Only delete this file when it matches the original Flutter template.
     const String legacyBuildSettingsWorkspace = '''
-<?xml version="1.0" encoding="UTF-8"?>	
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
-<plist version="1.0">	
-<dict>	
-	<key>BuildSystemType</key>	
-	<string>Original</string>	
-</dict>	
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
+	<string>Original</string>
+</dict>
 </plist>''';
 
     // contains instead of equals to ignore newline file ending variance.

--- a/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
@@ -7,9 +7,9 @@ import '../../base/logger.dart';
 import '../../project.dart';
 import 'ios_migrator.dart';
 
-/// Xcode legacy build system no longer supported by Xcode.
-/// Set in https://github.com/flutter/flutter/pull/21901/.
-/// Removed in https://github.com/flutter/flutter/pull/33684.
+// Xcode legacy build system no longer supported by Xcode.
+// Set in https://github.com/flutter/flutter/pull/21901/.
+// Removed in https://github.com/flutter/flutter/pull/33684.
 class XcodeBuildSystemMigration extends IOSMigrator {
   XcodeBuildSystemMigration(
     IosProject project,

--- a/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
@@ -42,7 +42,7 @@ class XcodeBuildSystemMigration extends IOSMigrator {
       logger.printStatus('Legacy build system detected, removing ${_xcodeWorkspaceSharedSettings.path}');
       _xcodeWorkspaceSharedSettings.deleteSync();
     }
-    
+
     return true;
   }
 }

--- a/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
+++ b/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart
@@ -1,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/logger.dart';
+import '../../project.dart';
+import 'ios_migrator.dart';
+
+/// Xcode legacy build system no longer supported by Xcode.
+/// Set in https://github.com/flutter/flutter/pull/21901/.
+/// Removed in https://github.com/flutter/flutter/pull/33684.
+class XcodeBuildSystemMigration extends IOSMigrator {
+  XcodeBuildSystemMigration(
+    IosProject project,
+    Logger logger,
+  ) : _xcodeWorkspaceSharedSettings = project.xcodeWorkspaceSharedSettings,
+      super(logger);
+
+  final File _xcodeWorkspaceSharedSettings;
+
+  @override
+  bool migrate() {
+    if (!_xcodeWorkspaceSharedSettings.existsSync()) {
+      logger.printTrace('Xcode workspace settings not found, skipping migration');
+      return true;
+    }
+
+    final String contents = _xcodeWorkspaceSharedSettings.readAsStringSync();
+
+    // Only delete this file when it matches the original Flutter template.
+    const String legacyBuildSettingsWorkspace = '''
+<?xml version="1.0" encoding="UTF-8"?>	
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
+<plist version="1.0">	
+<dict>	
+	<key>BuildSystemType</key>	
+	<string>Original</string>	
+</dict>	
+</plist>''';
+
+    // contains instead of equals to ignore newline file ending variance.
+    if (contents.contains(legacyBuildSettingsWorkspace)) {
+      logger.printStatus('Legacy build system detected, removing ${_xcodeWorkspaceSharedSettings.path}');
+      _xcodeWorkspaceSharedSettings.deleteSync();
+    }
+    
+    return true;
+  }
+}

--- a/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
+++ b/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
@@ -12,6 +12,7 @@ import 'package:platform/platform.dart';
 import 'package:flutter_tools/src/base/logger.dart';
 import 'package:flutter_tools/src/base/terminal.dart';
 import 'package:flutter_tools/src/ios/migrations/remove_framework_link_and_embedding_migration.dart';
+import 'package:flutter_tools/src/ios/migrations/xcode_build_system_migration.dart';
 import 'package:flutter_tools/src/macos/xcode.dart';
 import 'package:flutter_tools/src/project.dart';
 import 'package:flutter_tools/src/reporting/reporting.dart';
@@ -255,6 +256,84 @@ keep this 2
         );
         expect(() =>iosProjectMigration.migrate(), throwsToolExit(message: 'Your Xcode project requires migration'));
         verify(mockUsage.sendEvent('ios-migration', 'remove-frameworks', label: 'failure', value: null));
+      });
+    });
+
+    group('new Xcode build system', () {
+      MemoryFileSystem memoryFileSystem;
+      BufferLogger testLogger;
+      MockIosProject mockIosProject;
+      File xcodeWorkspaceSharedSettings;
+
+      setUp(() {
+        memoryFileSystem = MemoryFileSystem();
+        xcodeWorkspaceSharedSettings = memoryFileSystem.file('WorkspaceSettings.xcsettings');
+
+        testLogger = BufferLogger(
+          terminal: AnsiTerminal(
+            stdio: null,
+            platform: const LocalPlatform(),
+          ),
+          outputPreferences: OutputPreferences.test(),
+        );
+
+        mockIosProject = MockIosProject();
+        when(mockIosProject.xcodeWorkspaceSharedSettings).thenReturn(xcodeWorkspaceSharedSettings);
+      });
+
+      testWithoutContext('skipped if files are missing', () {
+        final XcodeBuildSystemMigration iosProjectMigration = XcodeBuildSystemMigration(
+          mockIosProject,
+          testLogger,
+        );
+        expect(iosProjectMigration.migrate(), isTrue);
+
+        expect(xcodeWorkspaceSharedSettings.existsSync(), isFalse);
+
+        expect(testLogger.traceText, contains('Xcode workspace settings not found, skipping migration'));
+        expect(testLogger.statusText, isEmpty);
+      });
+
+      testWithoutContext('skipped if nothing to upgrade', () {
+        const String contents = '''
+<?xml version="1.0" encoding="UTF-8"?>	
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
+<plist version="1.0">	
+<dict>	
+	<key>BuildSystemType</key>	
+	<string></string>	
+</dict>	
+</plist>''';
+        xcodeWorkspaceSharedSettings.writeAsStringSync(contents);
+
+        final XcodeBuildSystemMigration iosProjectMigration = XcodeBuildSystemMigration(
+          mockIosProject,
+          testLogger,
+        );
+        expect(iosProjectMigration.migrate(), isTrue);
+        expect(testLogger.statusText, isEmpty);
+      });
+
+      testWithoutContext('Xcode project is migrated', () {
+        const String contents = '''
+<?xml version="1.0" encoding="UTF-8"?>	
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
+<plist version="1.0">	
+<dict>	
+	<key>BuildSystemType</key>	
+	<string>Original</string>	
+</dict>	
+</plist>''';
+        xcodeWorkspaceSharedSettings.writeAsStringSync(contents);
+
+        final XcodeBuildSystemMigration iosProjectMigration = XcodeBuildSystemMigration(
+          mockIosProject,
+          testLogger,
+        );
+        expect(iosProjectMigration.migrate(), isTrue);
+        expect(xcodeWorkspaceSharedSettings.existsSync(), isFalse);
+
+        expect(testLogger.statusText, contains('Legacy build system detected, removing'));
       });
     });
   });

--- a/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
+++ b/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
@@ -316,13 +316,13 @@ keep this 2
 
       testWithoutContext('Xcode project is migrated', () {
         const String contents = '''
-<?xml version="1.0" encoding="UTF-8"?>	
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
-<plist version="1.0">	
-<dict>	
-	<key>BuildSystemType</key>	
-	<string>Original</string>	
-</dict>	
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
+	<string>Original</string>
+</dict>
 </plist>''';
         xcodeWorkspaceSharedSettings.writeAsStringSync(contents);
 

--- a/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
+++ b/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
@@ -287,7 +287,6 @@ keep this 2
           testLogger,
         );
         expect(iosProjectMigration.migrate(), isTrue);
-
         expect(xcodeWorkspaceSharedSettings.existsSync(), isFalse);
 
         expect(testLogger.traceText, contains('Xcode workspace settings not found, skipping migration'));
@@ -311,6 +310,7 @@ keep this 2
           testLogger,
         );
         expect(iosProjectMigration.migrate(), isTrue);
+        expect(xcodeWorkspaceSharedSettings.existsSync(), isTrue);
         expect(testLogger.statusText, isEmpty);
       });
 

--- a/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
+++ b/packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart
@@ -296,13 +296,13 @@ keep this 2
 
       testWithoutContext('skipped if nothing to upgrade', () {
         const String contents = '''
-<?xml version="1.0" encoding="UTF-8"?>	
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">	
-<plist version="1.0">	
-<dict>	
-	<key>BuildSystemType</key>	
-	<string></string>	
-</dict>	
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
+	<string></string>
+</dict>
 </plist>''';
         xcodeWorkspaceSharedSettings.writeAsStringSync(contents);
 

